src/index.ts

```diff
@@ -330,7 +333,9 @@ function activateLatexPlugin(
     // Run an initial latexRequest so that the appropriate files exist,
     // then open them.
     onFileChanged().then(() => {
-      findOpenOrRevealPDF();
+	  if (!disableOpenOrReveal) {
```

Comment:
I think we could also do this by checking whether the error panel has been created. So this check can be
```typescript
if (!errorPanel) {
  findOpenOrRevealPDF();
}
```
This would avoid the need to create a new variable.